src/Queue/RabbitMQQueue.php

```diff
@@ -525,6 +525,11 @@ protected function createMessage($payload, int $attempts = 0): array
         }
 
         if (isset($currentPayload['data']['command'])) {
+            // If the command data is encrypted, decrypt it first before attempting to unserialize
+            if (in_array(ShouldBeEncrypted::class, class_implements($currentPayload['data']['commandName']))) {
+                $currentPayload['data']['command'] = decrypt($currentPayload['data']['command']);
```

Comment:
Let's add the actual import for the decrypt function here.

Reply:
@khepin will do :)

Reply:
@khepin Will do :)